runtime-common/search-index.ts

```diff
@@ -509,25 +467,13 @@ export class SearchIndex {
     return this.exportedCardRefs.get(module) ?? [];
   }
 
-  // TODO merge this into the .search() API after we get the queries working
-  async directory(
-    url: URL
-  ): Promise<{ name: string; kind: Kind }[] | undefined> {
-    return this.directories.get(url);
-  }
-
-  // TODO merge this into the .search() API after we get the queries working
-  async module(url: URL): Promise<string | undefined> {
-    return this.modules.get(url)?.src;
-  }
-
   private async getIgnorePatterns(
     url: URL
   ): Promise<{ content: string; lastModified: number } | undefined> {
     let ref = await this.readFileAsText(
       this.realmPaths.local(new URL(".monacoignore", url))
     );
-
+    // are these supposed to be mutually exclusive?
```

Comment:
We can get rid of the `.monacoignore` file and only use `.gitignore` to remove any confusion